rascal2/ui/model.py

```diff
@@ -27,3 +27,19 @@ def create_project(self, name: str, save_path: str):
         self.project = RAT.Project(name=name)
         self.controls = RAT.Controls()
         self.save_path = save_path
+
+    def update_project(self, problem_definition: RAT.rat_core.ProblemDefinition):
```

Comment:
When I run DE (default controls) it crashes for me here

```
Traceback (most recent call last):
  File "/home/stephen/Documents/RasCAL-2-5-9-run-and-console/rascal2/ui/presenter.py", line 98, in handle_results
    self.model.update_project(self.runner.updated_problem)
  File "/home/stephen/Documents/RasCAL-2-5-9-run-and-console/rascal2/ui/model.py", line 45, in update_project
    getattr(self.project, class_list)[index].value = value
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/stephen/miniconda3/envs/rascal2/lib/python3.12/site-packages/pydantic/main.py", line 853, in __setattr__
    self.__pydantic_validator__.validate_assignment(self, name, value)
pydantic_core._pydantic_core.ValidationError: 1 validation error for ProtectedParameter
  Value error, value 0.0 is not within the defined range: 1.0 <= value <= 5.0 [type=value_error, input_value=ProtectedParameter(name='...orm', mu=0.0, sigma=inf), input_type=ProtectedParameter]
    For further information visit https://errors.pydantic.dev/2.8/v/value_error
```

Reply:
this seems to be an API bug:
```python
import RATapi as RAT

RAT.run(RAT.Project(), RAT.Controls(procedures="de"))
```
will produce the same error. I assume this is something to do with the project being blank?

Reply:
I will open an issue the problem also exist in the MATLAB